app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,292 @@
+    <string name="title_activity_result_contact">Contatto trovato</string>
+    <string name="action_help">Aiuto</string>
+
+    <!-- ### WELCOME DIALOG ### -->
+    <string name="welcome_text">L\'app PRIVACY FRIENDLY QR Code Scanner consente di scansionare codici a barre e codici-QR di varia tipologia richiedendo un numero minimo di permessi e rilevando collegamenti dannosi.</string>
+    <string name="welcome">Benvenuto</string>
+    <string name="viewhelp">Visualizza Aiuto</string>
+
+    <!--<string name="help_privacy_heading">Privacy Info</string>
+    <string name="about_author_contributors">Contributors</string>
+    <string name="privacy_friendly">This application belongs to the group of PRIVACY FRIENDLY Apps developed by Technische Universität Darmstadt. Sourcecode licensed under GPLv3. Images copyright TU Darmstadt and Google Inc.</string>
+    <string name="more_info">More information can be found on:</string>
+    <string name="github" translatable="false"><a href="https://github.com/SecUSo/privacy-friendly-qr-scanner">Github-Repo</a></string>-->
+    <string name="help_permission1_description">L\'applicazione richiede il permesso di accedere alla fotocamera, che viene utilizzato per scansionare i codici a barre ed i codici QR attraverso la fotocamera.</string>
+    <string name="help_permission1_heading">Fotocamera</string>
+    <string name="help_privacy_heading">Quali permessi sono richiesti da PRIVACY FRIENDLY QR Scanner?</string>
+    <string name="help_category_example">La scansione non funziona, cosa posso fare?</string>
+    <string name="help_overview_heading">Che cos\'è PRIVACY FRIENDLY QR Scanner?</string>
+    <string name="help_intro">Con PRIVACY FRIENDLY QR Scanner puoi scansionare codici a barre e codici QR di varia tipologia.</string>
+    <string name="help_whatis_answer">Con PRIVACY FRIENDLY QR Scanner puoi scansionare e generare codici a barre e codici QR di varia tipologia. Inoltre puoi copiare e condividere i codici QR scansionati con altre applicazioni, e salvare i codici QR generati nella galleria e condividerli.</string>
+
+
+    <!--###HELP###-->
+
+
+    <string name="help_whatis">Che cos\'è PRIVACY FRIENDLY QR Scanner?</string>
+
+    <string name="help_usability">La scansione non funziona, cosa posso fare?</string>
+    <string name="help_usability_answer">Se non puoi scansionare un codice, aumenta la distanza tra lo scanner ed il codice.</string>
+    <string name="help_privacy">Cosa rende l\'app privacy-friendly?</string>
+    <string name="help_privacy_answer">L\'app Privacy Friendly non richiede alcun permesso. Inoltre non include alcun meccanismo di tracking od annuncio.</string>
+    <string name="help_permission">Quali permessi sono richiesti da PRIVACY FRIENDLY QR Scanner?</string>
+    <string name="help_permission_answer">L\'applicazione richiede il permesso di accedere alla fotocamera, che viene utilizzato per scansionare i codici a barre ed i codici QR attraverso la fotocamera.</string>
```

Comment:
utilizzata

Reply:
"che viene utilizzato", we are talking about the permission, hence, "utilizzato"

Do you agree?

Reply:
hold on, i know how to fix it

Reply:
```L\'applicazione richiede il permesso di accedere alla fotocamera, utilizzata per scansionare i codici a barre ed i codici QR.```

Reply:
That would have been my proposal. perfect.